src/lib/y2storage/fake_device_factory.rb

```diff
@@ -581,7 +571,7 @@ def create_lvm_lv_stripe_parameters(lvm_lv, args)
       lvm_lv.stripes = stripes if stripes > 0
 
       stripe_size = args["stripe_size"] || DiskSize.zero
-      lvm_lv.stripe_size = stripe_size.size if stripe_size.size > 0
+      lvm_lv.stripe_size = stripe_size if stripe_size.to_i > 0
```

Comment:
fine for me. Btw maybe having method `zero?` make sense here. I do not see better name for "empty size", but it looks like quite common case to check it.